loqusdb/build_models/variant.py

```diff
@@ -15,18 +15,17 @@
 # These are coordinate for the pseudo autosomal regions in GRCh37
 
 
-def check_par(chrom, pos, genome_build=None):
+def check_par(chrom, pos, genome_build):
```

Comment:
Indeed, seems hard to get around that GRCh37 default from the main cli. I guess the advantage of setting the build optional (defaulting to something) would be you would not have to be explicit about it in the tests. But I think I see why you made that call. Better to avoid the sense that a proper default has to be set every time build is touched elsewhere.